### PR TITLE
Feature/hdfs users

### PR DESCRIPTION
**Pull request by OleksandrBerchenko**

### CODE REVIEW

**Comment by OleksandrBerchenko** [comment]
@Y4Rv1K, please take a look.

**Comment by Y4Rv1K** [comment]
@OleksandrBerchenko I see that it already not valid?

**Comment by OleksandrBerchenko** [comment]
Hm... It's valid in fact :)

File: roles/hdfs-standalone/defaults/main.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ hdfs_version: "2.6.0"
 hdfs_conf_dir: "/etc/hadoop/conf"
 hdfs_install_dir: "/usr/local/share"
 hdfs_namenode_port: 8020
+hdfs_namenode_host: ""
```
Comment (Y4Rv1K) [inline roles/hdfs-standalone/defaults/main.yml]: I would prefer use 'localhost' as dummy and change it on higher level if there are needed.
TBD

**Comment by Y4Rv1K** [comment]
Looks ok. Please proceed with testing

**Pull request closed without merging**